pkg/components/components.go

```diff
@@ -308,7 +309,30 @@ func GetDeployment(version string, operatorVersion string, namespace string, rep
 									Drop: []corev1.Capability{corev1.Capability("ALL")},
 								},
 							},
-							TerminationMessagePolicy: corev1.TerminationMessageFallbackToLogsOnError,
+							LivenessProbe: &corev1.Probe{
+								ProbeHandler: corev1.ProbeHandler{
+									HTTPGet: &corev1.HTTPGetAction{
+										Path: "/healthz",
+										Port: intstr.FromString("healthprobe"),
+									},
+								},
+								InitialDelaySeconds: 15,
+								PeriodSeconds:       20,
```

Comment:
those are known best practice i guess ? (same for the ones below)

Reply:
Yep